### ISSUE TITLE
Accessibility problem, voiceover and editing the default favourite bookmarks

### ISSUE THREAD

**Issue by ghost** [brave/brave-ios#2183]
### Description: 
Voiceover cannot edit the default favourite websites  
 

### Steps to Reproduce 
  1. create a new tab
  2. long press either icon, e.g. youtube
  3. voiceover cannot locate any edit buttons or edit functions
 

**Actual result:** 
after long press, voiceover didn't aware any change;
double tab the icon will open the webpage directly 

**Expected result:**
can use voiceover gesture to edit and delete the default favourite
 
**Reproduces how often:** [Easily reproduced, Intermittent Issue]
everytime

**Brave Version:** 
1.14.2 版 (19.12.12.17)

**Device details:** 
iphone 8 ios 13.3

**Comment by kylehickinson** [comment]
This should function correctly now with the new iOS 13 context menus being used in place of a custom edit button 🙂
